packages/app/src/components/DeleteCredentialSheet.tsx

```diff
@@ -0,0 +1,84 @@
+import { usePidCredential } from '@easypid/hooks'
+import { type CredentialForDisplayId, deleteCredential, useAgent } from '@package/agent/src'
+import { Heading, HeroIcons, Paragraph, Stack, XStack, useToastController } from '@package/ui'
+import { FloatingSheet } from '@package/ui/src/panels/FloatingSheet'
+import { useState } from 'react'
+import { useHaptics } from '../hooks'
+import { DualResponseButtons } from './DualResponseButtons'
+
+interface DeleteCredentialSheetProps {
+  isSheetOpen: boolean
+  setIsSheetOpen: (isOpen: boolean) => void
+  id: CredentialForDisplayId
+  name: string
+}
+
+export function DeleteCredentialSheet({ isSheetOpen, setIsSheetOpen, id, name }: DeleteCredentialSheetProps) {
+  const { credential } = usePidCredential()
+  const toast = useToastController()
+  const { agent } = useAgent()
+  const [isLoading, setIsLoading] = useState(false)
+  const { withHaptics, error, success } = useHaptics()
```

Comment:
I was kinda confused where the `success` and `error` methods came from. I think we should call them `successHaptics` and `errorHaptics`. Then it's immediatley clear